templates/frag/operation.hbs

```diff
@@ -150,6 +174,9 @@ open func {{{name}}}Request({{{_params}}}) throws -> URLRequest {
     
 {{/with}}
 {{/if}}
+
+    {{>frag/requestSecurity}}
```

Comment:
I wonder if this should also be after the addition of headers... so it's basically the final word on the subject of the request except for...?